Resources/doc/reference/introduction.rst

```diff
@@ -73,6 +73,9 @@ By default the ``SonataPageBundle`` is shipped with core block services:
 
  - sonata.page.block.container      : Block container
  - sonata.page.block.children_pages : Render a navigation panel
+ - sonata.page.block.breadcrumb     : Render a breadcrumb
```

Comment:
OK. I think the doc would be richer if the 5 block services existing in the page bundle were listed here (as proposed), no ?